### DISCUSSION TITLE
Any plans to create subgroups of active jobs to (running,failed,funished)

### DISCUSSION THREAD

**Discussion by ShingaJ** [apache/airflow#30573] [Q&A]
Is there any plan to create a subgroup of active jobs, to drill down to see running, failed, and finished dags,

**Comment by potiuk** [comment]
I believe there is PR in progress at least for part of it https://github.com/apache/airflow/pull/30429 - and you can help there by providing helpful suggestions, reviews and generally providing encouraging feedback and offering to help with testing. The usual thinng - when you want to speed up getting something you would like to see, spending your own time on helping to make things happen generally speeds up things.
